lib/solidus_dev_support/rspec/coverage.rb

```diff
@@ -19,6 +19,14 @@
 if ENV['CODECOV_TOKEN']
   require 'codecov'
   SimpleCov.formatter = SimpleCov::Formatter::Codecov
+  warn <<~WARN
+    DEPRECATION WARNING: The Codecov ruby uploader is deprecated.
+    Please use the Codecov CLI uploader to upload code coverage reports.
+    See https://docs.codecov.com/docs/deprecated-uploader-migration-guide#ruby-uploader for more information on upgrading.
+  WARN
+elsif ENV['CODECOV_COVERAGE_PATH']
+  require 'simplecov-cobertura'
+  SimpleCov.formatter = SimpleCov::Formatter::CoberturaFormatter
 else
-  warn "Provide a CODECOV_TOKEN environment variable to enable Codecov uploads"
+  warn "Provide either a CODECOV_TOKEN or CODECOV_COVERAGE_PATH environment variable to enable Codecov uploads"
```

Comment:
If it's deprecated I think we can drop the suggestion to use CODECOV_TOKEN, it can survive in the git history and that's it, wdyt?

Reply:
yeah, great idea